dhall/tests/format/applicationMultilineA.dhall

```diff
@@ -18,7 +18,7 @@
 , someList =
     Some
       [ aaaaaaaaaaaaaaaaaaaaaaaaaa, bbbbbbbbbbbbbbbbbbbb, cccccccccccccccccccccccc, dddddddddddddd ]
-, merge =
+, merge1 =
```

Comment:
For future reference, another thing you can do to keep the original name is to quote the label in backticks:

```dhall
, `merge` =
```

... but this is fine, too